designer/client/src/components/graph/node-modal/editors/expression/FixedValuesEditor.tsx

```diff
@@ -46,6 +51,7 @@ export const FixedValuesEditor: ExtendedEditor<Props> = (props: Props) => {
     };
 
     const { expressionObj, readOnly, onValueChange, className, showValidation, editorConfig, fieldErrors } = props;
+    const mode = FixedValuesEditorMode[editorConfig.mode || "LIST"];
```

Comment:
What's the reason for introducing editor mode concept instead of adding a new editor

Reply:
For a brief moment it seemed to be an easier approach. Fixed by adding a new editor.